optuna/importance/_fanova/_fanova.py

```diff
@@ -80,51 +76,35 @@ def fit(
             # This could occur if for instance `X.shape[0] == 1`.
             raise RuntimeError("Encountered zero total variance in all trees.")
 
-    def get_importance(self, features: Tuple[int, ...]) -> Tuple[float, float]:
+    def get_importance(self, feature: int) -> Tuple[float, float]:
         # Assert that `fit` has been called.
         assert self._trees is not None
-        assert self._variances is not None
 
-        self._compute_variances(features)
+        self._compute_variances(feature)
 
         fractions: Union[List[float], numpy.ndarray] = []
 
         for tree_index, tree in enumerate(self._trees):
             tree_variance = tree.variance
             if tree_variance > 0.0:
-                fraction = self._variances[features][tree_index] / tree_variance
+                fraction = self._variances[feature][tree_index] / tree_variance
                 fractions = numpy.append(fractions, fraction)
 
         fractions = numpy.asarray(fractions)
 
         return float(fractions.mean()), float(fractions.std())
 
-    def _compute_variances(self, features: Tuple[int, ...]) -> None:
+    def _compute_variances(self, feature: int) -> None:
         assert self._trees is not None
-        assert self._variances is not None
         assert self._column_to_encoded_columns is not None
 
-        if features in self._variances:
+        if feature in self._variances:
             return
 
-        for k in range(1, len(features)):
-            for sub_features in itertools.combinations(features, k):
-                if sub_features not in self._variances:
-                    self._compute_variances(sub_features)
-
-        raw_features = numpy.concatenate([self._column_to_encoded_columns[f] for f in features])
-
+        raw_feature = self._column_to_encoded_columns[feature]
```

Comment:
A nit but how about keeping it plural? I think it's more natural given that it's still an ndarray with possibly more than 1 element (for categorical with more than 1 choice). It's still also the name of the argument in `get_marginal_variance`.
```suggestion
        raw_features = self._column_to_encoded_columns[feature]
```

Reply:
Thank you for your review. Yes, it may contain multiple values :bow: I renamed it at 
72b564a.